### PR TITLE
Add scss source maps

### PR DESCRIPTION
**Pull request by maxxcrawford**
This PR solves a request from @heaversm. This should make debugging CSS issues easier! 

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,6 @@
     "cpr": "3.0.1",
     "csurf": "1.11.0",
     "del": "^6.0.0",
-    "dotenv": "8.2.0",
```
Comment (toufali) [inline package.json]: Wondering why dotenv was removed?  I think it's still in use?  🤔

**Review by toufali — COMMENTED** [review]
Haven't tested locally yet, but changes seem straightforward apart from the `dotenv` removal that I commented on 👍

File: package.json (hunk 1 above)
Comment (maxxcrawford) [reply package.json]: Already added it back. 🙈  f872ff7

Comment (maxxcrawford) [reply package.json]: I was using it conditional write the sourcemap in the Gulpfile and it got sticky, so I bailed. I had initally (installed) `dotenv`, and didnt' realize it was already in the package.json/in use.

Comment (maxxcrawford) [reply package.json]:
```
┬─┬ノ(ಠ_ಠノ)
```

**Review by toufali — APPROVED** [review]

**Comment by heaversm** [comment]
FWIW I've been using this in my own branch and it seems to work great.

**Pull request merged**